Backend/tests/test_user_ctrl.py

```diff
@@ -327,6 +327,20 @@ def test_joinClass(self):
     #confirm none joined
     self.assertFalse(joined)
 
+  def test_resetPassword(self):
```

Comment:
Technically all methods in `user_ctrl` that are written should be unit tested. Does this hit all branches for other methods?